client/js/actions/view-action-creator.js

```diff
@@ -17,11 +17,11 @@ var ViewActionCreator = {
 
     ApiUtils.getDayData(date, ServerActionCreator.recieveDayData);
   },
-  getMonthEvents: function (date) {
+  getMonthData: function (date) {
     AppDispatcher.dispatch({
-      actionType: AppConstants.ViewActionTypes.GET_MONTH_EVENTS
+      actionType: AppConstants.ViewActionTypes.GET_MONTH_Data
```

Comment:
i think that you may have forgotten to add the GET_MONTH_DATA constant to the constants file